docs/advanced-features/custom-directories.md

```diff
@@ -0,0 +1,42 @@
+---
+description: Customize the directories included in your Next.js project.
+---
+
+# Custom Directories
+
+If you need to customize which directories are included in your project, i.e. to
+prevent redundant type-checking during `next build`, you can adjust the
+`include` and `exclude` fields of your root-level tsconfig.json to do so:
+
+```none
+{
+  "exclude": ["excludeDir/"],
```

Comment:
You could provide real-life examples, like "cypress/", "vendors/", ".storybook" etc.